plugins/events-node/api-report.md

```diff
@@ -0,0 +1,44 @@
+## API Report File for "@backstage/plugin-events-node"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { ExtensionPoint } from '@backstage/backend-plugin-api';
+
+// @public
+export interface EventBus {
+  publish(topic: string, event: unknown): Promise<void>;
+  subscribe(
+    ...subscribers: Array<EventSubscriber | Array<EventSubscriber>>
+  ): void;
+}
+
+// @public
+export interface EventPublisher {
+  // (undocumented)
+  connect(eventBus: EventBus): Promise<void>;
+}
+
+// @alpha (undocumented)
```

Comment:
I wonder where the limit should go for having things be alpha or not at this stage. I'm honestly not sure. Since the packages are brand new, maybe it doesn't make a lot of difference?

Reply:
you are right. In the beginning I though it is still very new stuff, etc. however that is also reflected in the version anyways. I will change it.

Reply:
I kept it alpha as all things related to the new backend plugin api were marked as alpha. If you think it is ok to remove it I can change it.

Reply:
Nah let's keep it like this for now.